ulauncher/main.py

```diff
@@ -39,6 +39,11 @@ def setup(self, _):
         self.window.set_application(self)
         self.window.finish_initializing()
 
+    def do_before_emit(self, data ):
+        query = data.lookup_value("query",  GLib.VariantType("s"))
+        if query:      
+           self.window.initial_query = query.unpack() + " "
```

Comment:
I think we shouldn't add the space. If people want the space they can send ulauncher-toggle -q "keyword ". If people don't want a space they can't send negative one character.

Reply:
Yes, makes sense!